nss.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: nss
+  version: 3.94
+  epoch: 0
+  description: "Network Security Services (NSS) is a set of libraries designed to support cross-platform development of security-enabled client and server applications."
+  copyright:
+    - license: MPL-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - mercurial
+      - bash
+      - py3-gyp-next
+      - samurai
+      - perl
+      - zlib-dev
+      - libnspr
+      - libnspr-dev
+      - sqlite-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: nss-package-version
+
+pipeline:
+  # we need to use runs because melange dont have a mercurial checkout pipeline
+  - runs: |
+      hg clone -r NSS_${{vars.nss-package-version}}_RTM https://hg.mozilla.org/projects/nss
```

Comment:
I'd split this into its own first step.

NB: As these are growing, we should consider adding a pipeline to checkout mercurial.  You can see examples of custom pipelines in `pipelines/`